src/components/app-configure/general/AppBranding.vue

```diff
@@ -0,0 +1,192 @@
+<script lang="ts" setup>
+import { computed, reactive, type ComputedRef } from 'vue'
+import { useStore } from 'vuex'
+
+import SettingCard from '@/components/app-configure/SettingCard.vue'
+import VDropdown from '@/components/lib/VDropdown/VDropdown.vue'
+import VFileUpload from '@/components/lib/VFileUpload/VFileUpload.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+import { availableThemes } from '@/utils/constants'
+
+type Theme = {
+  label: string
+  value: string
+}
+
+const store = useStore()
+
+type ThemeLogoOption = {
+  logo: File | undefined
+  horizontalLogo: File | undefined
+}
+
+type ThemeLogo = {
+  dark: ThemeLogoOption
+  light: ThemeLogoOption
+}
+
+const themeLogos: ThemeLogo = reactive({
+  dark: {
+    logo: undefined,
+    horizontalLogo: undefined,
+  },
+  light: {
+    logo: undefined,
+    horizontalLogo: undefined,
+  },
+})
```

Comment:
Let's make this a prop and allow for the case when a URL is returned from the API.

Reply:
Using values from store instead of making these as props